dockstore-webservice/src/main/java/io/dockstore/webservice/core/VersionMetadata.java

```diff
@@ -0,0 +1,66 @@
+/*
+ *    Copyright 2019 OICR
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package io.dockstore.webservice.core;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.MapsId;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+/**
+ * Data about versions of a workflow/tool in Dockstore rather than about the original workflow. Stays non-immutable
```

Comment:
This is a sensible class to use as a member for versions. The justification for this change is to separate the mutable and immutable metadata on an entry's version. This policy could be made more clear for subsequent programmers by expounding on "Stays non-immutable" a little.